src/lib/auth.ts

```diff
@@ -20,7 +21,18 @@ const writePerms = (
 
 const matchesActor = 'actor eq @__ACTOR_ID';
 const matchesUser = `user/any(u:u/${matchesActor})`;
-const ownsDevice = `owns__device/any(d:d/${matchesActor})`;
+const matchesNonFrozenDeviceActor = (alias: string) => {
```

Comment:
You could default to no alias, eg
```suggestion
const matchesNonFrozenDeviceActor = (alias = '') => {
```